scripting/get5/matchconfig.sp

```diff
@@ -208,9 +208,8 @@ stock bool LoadMatchFromUrl(const char[] url, ArrayList paramNames = null,
   strcopy(g_LoadedConfigUrl, sizeof(g_LoadedConfigUrl), cleanedUrl);
 
   if (steamWorksAvaliable) {
-    // Add the protocl strings. Only allow http since SteamWorks doesn't support http it seems?
-    ReplaceString(cleanedUrl, sizeof(cleanedUrl), "https://", "http://");
-    if (StrContains(cleanedUrl, "http://") == -1) {
+    // Add the protocol strings if missing (only http).
+    if (StrContains(cleanedUrl, "http://") == -1 && StrContains(cleanedUrl, "https://") == -1) {
```

Comment:
This expression seems like it will never be true anymore, right? 

To be true it would have to contain both `http://` and `https://`, which seems largely impossible for any sane url.

Reply:
`StrContains()` returns `-1` when not found or the index of the match otherwise. Therefore, `http://` will be added if there is neither http nor https 😁

Reply:
@Drarig29 is right :) We have also tested this part of the codechange.

Reply:
Ah, yes, I didn't read carefully enough - I hate that SM's `StrContains` doesn't return a bool. Thanks